<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow4</class>
 <widget class="QMainWindow" name="MainWindow4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(32, 32, 32)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>341</width>
      <height>371</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px;</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="3">
      <widget class="QPushButton" name="button_pi">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 15px;
background-color: rgb(59, 60, 63);

</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup1</string>
       </attribute>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QPushButton" name="button_equal">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 30px;
background-color:rgb(38, 186, 217);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="button_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 20px;
background-color:rgb(50, 50, 50);
</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="button_pm">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 20px;
background-color:rgb(50, 50, 50);
</string>
       </property>
       <property name="text">
        <string>+/-</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="button_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 20px;
background-color:rgb(50, 50, 50);
</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="button_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 20px;
background-color:rgb(50, 50, 50);
</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="button_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 20px;
background-color:rgb(50, 50, 50);
</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="button_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 20px;
background-color:rgb(50, 50, 50);
</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="button_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 20px;
background-color:rgb(50, 50, 50);
</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="button_deg">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 15px;
background-color: rgb(59, 60, 63);

</string>
       </property>
       <property name="text">
        <string>deg</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup1</string>
       </attribute>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="button_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 20px;
background-color:rgb(50, 50, 50);
</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="button_cos">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 15px;
background-color: rgb(59, 60, 63);

</string>
       </property>
       <property name="text">
        <string>cos</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup1</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="button_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 20px;
background-color:rgb(50, 50, 50);
</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="button_cosh">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 15px;
background-color: rgb(59, 60, 63);

</string>
       </property>
       <property name="text">
        <string>cosh</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup1</string>
       </attribute>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="button_0">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 20px;
background-color:rgb(50, 50, 50);
</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="button_log">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string>font-size: 15px;
background-color: rgb(59, 60, 63);

</string>
       </property>
       <property name="text">
        <string>log</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup1</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="button_sin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 15px;
background-color: rgb(59, 60, 63);

</string>
       </property>
       <property name="text">
        <string>sin</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup1</string>
       </attribute>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="button_tanh">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 15px;
background-color: rgb(59, 60, 63);

</string>
       </property>
       <property name="text">
        <string>tanh</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup1</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="button_nlog">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string>font-size: 15px;
background-color: rgb(59, 60, 63);

</string>
       </property>
       <property name="text">
        <string>ln</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup1</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="button_cosin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 15px;
background-color: rgb(59, 60, 63);

</string>
       </property>
       <property name="text">
        <string>cos-1</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup1</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="button_sinh">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 15px;
background-color: rgb(59, 60, 63);

</string>
       </property>
       <property name="text">
        <string>sinh</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup1</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="button_tan">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 15px;
background-color: rgb(59, 60, 63);

</string>
       </property>
       <property name="text">
        <string>tan</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup1</string>
       </attribute>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QPushButton" name="button_dot">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 20px;
background-color:rgb(50, 50, 50);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="button_sinin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 15px;
background-color: rgb(59, 60, 63);

</string>
       </property>
       <property name="text">
        <string>sin-1</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup1</string>
       </attribute>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="button_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 20px;
background-color:rgb(50, 50, 50);
</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="button_tanin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 15px;
background-color: rgb(59, 60, 63);

</string>
       </property>
       <property name="text">
        <string>tan-1</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup1</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="button_expo_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 15px;
background-color: rgb(59, 60, 63);

</string>
       </property>
       <property name="text">
        <string>e</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup1</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="button_expo_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 15px;
background-color: rgb(59, 60, 63);

</string>
       </property>
       <property name="text">
        <string>eˣ</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup1</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="button_clear_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 15px;
background-color: rgb(59, 60, 63);

</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup1</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="button_radian">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 15px;
background-color: rgb(59, 60, 63);

</string>
       </property>
       <property name="text">
        <string>rad</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="button_clear">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 15px;
background-color: rgb(59, 60, 63);

</string>
       </property>
       <property name="text">
        <string>C</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup1</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="button_factorial">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 15px;
background-color: rgb(59, 60, 63);

</string>
       </property>
       <property name="text">
        <string>x!</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup1</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>341</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 15px;
background-color: rgb(59, 60, 63);

</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>lineEdit</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCALCULATOR">
    <property name="title">
     <string>CALCULATOR</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSTANDARD"/>
    <addaction name="separator"/>
    <addaction name="actionCOMPLEX"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionLOG_TRIGO"/>
   </widget>
   <widget class="QMenu" name="menuOPTIONS">
    <property name="title">
     <string>OPTIONS</string>
    </property>
    <widget class="QMenu" name="menuTHEME">
     <property name="title">
      <string>THEME</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionDARK"/>
     <addaction name="separator"/>
     <addaction name="actionBRIGHT"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuTHEME"/>
    <addaction name="separator"/>
    <addaction name="actionRECORD"/>
    <addaction name="separator"/>
    <addaction name="actionEXIT"/>
    <addaction name="separator"/>
    <addaction name="actionABOUT"/>
   </widget>
   <addaction name="menuCALCULATOR"/>
   <addaction name="menuOPTIONS"/>
  </widget>
  <action name="actionSTANDARD">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/images/icons/simple.ico</normaloff>:/images/icons/simple.ico</iconset>
   </property>
   <property name="text">
    <string>STANDARD</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCOMPLEX">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/images/icons/iota.ico</normaloff>:/images/icons/iota.ico</iconset>
   </property>
   <property name="text">
    <string>COMPLEX</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionQUADRATIC">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/images/icons/quadratic.ico</normaloff>:/images/icons/quadratic.ico</iconset>
   </property>
   <property name="text">
    <string>QUADRATIC</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionLOG_TRIGO">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/images/icons/logarithm.ico</normaloff>:/images/icons/logarithm.ico</iconset>
   </property>
   <property name="text">
    <string>LOG&amp;TRIGO</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionRECORD">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/images/icons/record.ico</normaloff>:/images/icons/record.ico</iconset>
   </property>
   <property name="text">
    <string>RECORD</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionEXIT">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/images/icons/reject.ico</normaloff>:/images/icons/reject.ico</iconset>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionABOUT">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/images/icons/about.ico</normaloff>:/images/icons/about.ico</iconset>
   </property>
   <property name="text">
    <string>ABOUT</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionDARK">
   <property name="text">
    <string>DARK</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionBRIGHT">
   <property name="text">
    <string>BRIGHT</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup1">
   <property name="exclusive">
    <bool>true</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
